node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
Or we could introduce a new function like loc2. `tl.loc2(__dirname, 'resourceKey', formatArg1, formatArg2)` as it basically is an enhanced functionality. Discoverability will be tricky, but can be mitigated using intellisense.